oap-server/server-bootstrap/src/main/resources/component-libraries.yml

```diff
@@ -311,6 +311,9 @@ GraphQL:
 spring-annotation:
   id: 93
   languages: Java
+Hbase:
```

Comment:
I think all `Hbase` should be `HBase` according to the official document.

Reply:
I  fix  Htable--> HTable too, wait ci